### DISCUSSION TITLE
俄羅斯方塊對戰網站

### DISCUSSION THREAD

**Discussion by FakeWood** [1101-GS4719-C/Python-Community-GS4719-C#1591] [General]
相信大家都玩過俄羅斯方塊吧!
平常大家都是去哪裡玩呢?
這邊推薦大家一個可以線上對戰的網站：[Tetr.io](https://tetr.io/)
在這裡可以和世界各地的玩家交流，也可以創房間和朋友激戰，或是只是想自己一個人玩也沒問題！
進入網站後會先看到以下畫面
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/105701699/2f332fc6-c471-469f-8216-afaf13cfb77e)
若是想創建帳戶、紀錄成績的話就在這打入自己要取的帳戶名
或是只是想隨便玩玩的話，就隨便打個亂數
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/105701699/d6be670f-818e-47e8-b87e-874ec3c4c8ec)
然後看你是要註冊，還是要用訪客模式遊玩，點選相應的選項
就可以進入這個畫面了哦!
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/105701699/5323ecda-a4be-4189-90b8-5c17800e5b4c)
SOLO模式就是單機遊戲
MULTIPLAYER模式裡面還有多種模式可以選
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/105701699/fd36c5c7-efb9-4470-9897-95a6c5694ccf)
QUICK PLAYER，能夠跟其他線上玩家來場多人混戰，
CUSTOM GAME，可以跟朋友開房激戰

希望大家玩得愉快~

**Comment by TobywithDino** [comment]
超棒的分享！國小的時候都在電腦課和同學玩FB上面的俄羅斯方塊《Tetris Battle》，但在2019年5月左右，官方停止運營，大家都找不到這麼有趣又有競爭性的同款俄羅斯方塊遊戲，謝謝你找到這款遊戲，讓大家在電腦課不再無聊~
